Index: src/index.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from flask import Flask, render_template, request\r\nfrom src import app\r\n\r\nfrom  src.admin import *\r\n\r\n\r\n@app.route('/')\r\ndef home_page():\r\n    return render_template('index.html')\r\n\r\n\r\ndef admin_stats_page():\r\n    pass\r\n\r\n\r\nif __name__ == \"__main__\":\r\n    from src.admin import *\r\n    # debug to view debugging in the future\r\n    app.run(debug=True)
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/index.py b/src/index.py
--- a/src/index.py	(revision 5cfc33c3038a5c1ad06087f2f06cc0f31d36f9ae)
+++ b/src/index.py	(date 1639559710881)
@@ -1,7 +1,11 @@
 from flask import Flask, render_template, request
 from src import app
 
-from  src.admin import *
+from src.admin import *
+# app = Flask(__name__)
+
+
+# app = Flask(__name__)
 
 
 @app.route('/')
